wolfcrypt/src/random.c

```diff
@@ -2199,6 +2200,20 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
     }
 #endif
 
+int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
+{
+#ifdef WOLF_CRYPTO_CB
+    int ret;
+
+    if (os->devId != INVALID_DEVID) {
```

Comment:
Can we make sure and handle cases where `os` is NULL?

Reply:
fixed